packages/api/src/external/carequality/document/process-outbound-document-retrieval-resps.ts

```diff
@@ -46,8 +45,8 @@ export async function processOutboundDocumentRetrievalResps({
 
       await setDocQueryProgress({
         patient: { id: patientId, cxId: cxId },
-        downloadProgress: { status: "completed" },
-        convertProgress: { status: "completed" },
+        downloadProgress: { total: 0, status: "completed" },
+        convertProgress: { total: 0, status: "completed" },
```

Comment:
We need to set this explicitly to 0 because when we do the cleanup it will see that the total doesnt added up to (succ + err) which are 0.